### DISCUSSION TITLE
Export C header

### DISCUSSION THREAD

**Discussion by VarLad** [rockorager/libvaxis#135] [General]
Hey, is it possible to export this library as a C header? 
I wonder how much of an effort would it take to get a C header out of a Zig library.
Would make it straightforward to integrate in other languages.

**Comment by rockorager** [comment]
I don't think it would be too much of an effort. Overall I am for it, but I would want to look through some existing projects which do this to make sure we get the design right (ghostty and cyber come to mind)

**Comment by rockorager** [comment]
(Note: I converted this to a discussion for now. I want to keep issues only for accepted features / bugs)
